PluginProcessor.cpp

```diff
@@ -167,8 +167,11 @@ bool EffectsPluginProcessor::isBusesLayoutSupported (const AudioProcessor::Buses
 
 void EffectsPluginProcessor::processBlock (juce::AudioBuffer<float>& buffer, juce::MidiBuffer& /* midiMessages */)
 {
-    if (runtime == nullptr)
+    if (runtime == nullptr) {
```

Comment:
This is great, but I'm realizing two things as I read:
* I believe we can just use `buffer.clear()` instead of manually clearing each channel
* There's another case we should check for, where the `runtime` instance has been initialized but we have not yet applied the first set of instructions.

So I'm thinking we change this to:

```
void EffectsPluginProcessor::processBlock (...stuff...)
{
    scratchBuffer.makeCopyOf(buffer, true);
    buffer.clear();

   if (runtime != nullptr) {
       runtime->process(
           ...other...stuff...
       );
   }
}
```

This way the output buffer is always silent if the runtime is not writing into it (i.e. even in the case where the runtime exists but hasn't received instructions yet)

Look ok to you?

Reply:
Yep, that shouldn't be a problem